docs/developers/optimism-devnet.md

```diff
@@ -21,7 +21,7 @@ Next, clone the repo:
 cd $HOME
 git clone https://github.com/celestiaorg/optimism
 cd optimism
-git checkout v0.1.0-OP_v1.0.6-CN_v0.9.5
+git checkout tags/v0.1.1-OP_v1.0.6-CN_v0.11.0-rc1
```

Comment:
```suggestion
git checkout v0.1.1-OP_v1.0.6-CN_v0.11.0-rc1
```

Reply:
is this just cleaner? i was keeping it in line with other instances of `git checkout` in docs, but happy to add suggestion 👍